Alpaca.Markets/RestClient.General.cs

```diff
@@ -76,6 +76,45 @@ public sealed partial class RestClient
             }
         }
 
+        /// <summary>
+        /// Gets list of account activities from Alpaca REST API endpoint.
+        /// </summary>
+        /// <param name="activityTypes">The activity type you want to view entries for.</param>
+        /// <param name="date">The date for which you want to see activities.</param>
+        /// <param name="until">The response will contain only activities submitted before this date. (Cannot be used with date.)</param>
+        /// <param name="after">The response will contain only activities submitted after this date. (Cannot be used with date.)</param>
+        /// <param name="direction">The response will be sorted by time in this direction. (Default behavior is descending.)</param>
+        /// <param name="pageSize">The maximum number of entries to return in the response.</param>
+        /// <param name="pageToken">The ID of the end of your current page of results.</param>
+        /// <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
+        /// <returns>Read-only list of asset information objects.</returns>
+        public Task<IReadOnlyList<IAsset>> ListAccountActivitiesAsync(
+            IEnumerable<AccountActivityType> activityTypes = null,
+            DateTime? date = null,
+            DateTime? until = null,
+            DateTime? after = null,
+            SortDirection? direction = null,
+            Int64? pageSize = null,
+            String pageToken = null,
+            CancellationToken cancellationToken = default)
+        {
+            var builder = new UriBuilder(_alpacaHttpClient.BaseAddress)
```

Comment:
I suggest adding some arguments checking logic. At least, we should check what if `date` argument provided we don't provide `untill` or `after` arguments too.

Reply:
Done.